SB - remove support for Gardener `adminkubeconfig` (#1312)
kyma-project/kyma-environment-broker: docs/user/05-60-kyma-bindings.md

}
```

The Broker returns a kubeconfig with a JWT token used as a user authentication mechanism. The token is generated using Kubernetes TokenRequest attached to a ServiceAccount, ClusterRole, and ClusterRoleBinding, all named `kyma-binding-{{binding_id}}`. Such an approach allows for modifying the permissions granted to the kubeconfig.

> You can specify the duration for which the generated kubeconfig is valid by providing the `expiration_seconds` in the `parameter` field of the request body. 

Reviewer: ```suggestion
To specify the duration for which the generated kubeconfig is valid, provide the **expiration_seconds** in the `parameter` object of the request body. 
```
